integration_test/support/migration.exs

```diff
@@ -80,6 +81,10 @@ defmodule Ecto.Integration.Migration do
       timestamps
     end
 
+    alter table(:users) do
```

Comment:
Renaming a column is not an "action" as defined in
http://www.postgresql.org/docs/9.4/static/sql-altertable.html

So it currently must be run in a separate alter table(:xxx) call.  Is something like that acceptable?  Or, should we detect it in the backend and ensure two multiple alter tables are executed when a rename is used.